website/docs/Conditions.md

```diff
@@ -0,0 +1,231 @@
+
+If we want to be more verbose we can use `Conditions.Condition`.
+The above and below examples are completely equivalent:
+
+```js
+const NFTOwnershipConfig = {
+    contractAddress: "0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D",
+    standardContractType: "ERC721",
+    chain:  "ethereum",
+    method: "ownerOf",
+    parameters: [
+      "5954"
+    ],
+    returnValueTest: {
+      comparator: "==",
+      value: ":userAddress"
+    }
+  }
+const NFTOwnership = new Conditions.Condition(NFTOwnershipConfig)
+```
+## `Conditions.ERC721Balance`
+`Conditions.ERC721Balance` is a shortcut for building conditions that test for ownership of at least one ERC721 token (NFT) within a collection.
+
+```js
+const NFTBalance = new Conditions.ERC721Balance({
+    contractAddress: '0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D',
+});
+```
+Alternatively:
+```js
+const NFTBalanceConfig = {
+    contractAddress: "0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D",
+    standardContractType: "ERC721",
+    chain:  "ethereum",
+    method: "balanceOf",
+    parameters: [
+      ":userAddress"
+    ],
+    returnValueTest: {
+      comparator: ">",
+      value: "0"
+    }
+  }
+
+const NFTBalance = new Conditions.Condition(NFTBalanceConfig)
+```
+
+## `Conditions.TimelockCondition`
+`Conditions.TimelockCondition` is a shortcut for building conditions that test against block height.
+
+```js
+const timelock = Conditions.TimelockCondition({
+  returnValueTest: {
+    comparator: '>',
+    value: '100'
+  }
+});
+```
+
+or:
+
+```js
+const timelockConfig = {
+    contractAddress: "",
+    standardContractType: "",
+    chain:  "ethereum",
+    method: "timelock",
+    returnValueTest: {
+      comparator: ">",
+      value: "100"
+    }
+  }
+const timelock = Conditions.Condition(timelockConfig);
+```
+
+## `Conditions.RpcCondition`
+`Conditions.RpcCondition` is a shortcut for building conditions that test against standard [RPC calls](https://ethereum.org/en/developers/docs/apis/json-rpc/)
+
+```js
+const const rpc = new Conditions.RpcCondition({
+  chain: 'ethereum',
+  method: 'eth_getBalance',
+  parameters: [
+      ":userAddress",
+      "latest"
+    ],
+  returnValueTest: {
+    comparator: ">=",
+    value: "10000000000000"
+  }
+});
+```
+or:
+```js
+const rpcConfig = {
+    contractAddress: "",
+    standardContractType: "",
+    chain:  "ethereum",
+    method: "eth_getBalance",
+    parameters: [
+      ":userAddress",
+      "latest"
+    ],
+    returnValueTest: {
+      comparator: ">=",
+      value: "10000000000000"
+    }
+  }
+const rpc = Conditions.Condition(rpcConfig);
+```
+
+
+## `Conditions.Condition`
+`Conditions.Condition` provides full control over the configuration of a Condition.
+It  takes parameters:
+
+- `contractAddress` is the public address of the contract we'd like to query.
+- `standardContractType` can take values from `ERC20, ERC721` and `ERC1155`. Alternatively, an ABI can be passed through if a non standard contract is being used.
```

Comment:
There is also `functionAbi` parameter (not defined in this list) which is what is used to specify the ABI.